src/profile-logic/sanitize.js

```diff
@@ -169,11 +170,23 @@ function sanitizeThreadPII(
   const markersToDelete = new Set();
   if (
     PIIToBeRemoved.shouldRemoveUrls ||
+    PIIToBeRemoved.shouldRemovePreferenceValues ||
     PIIToBeRemoved.shouldRemoveThreadsWithScreenshots.size > 0
   ) {
     for (let i = 0; i < markerTable.length; i++) {
       const currentMarker = markerTable.data[i];
 
+      // Remove the all the preference values, if the user wants that.
+      if (
+        PIIToBeRemoved.shouldRemovePreferenceValues &&
+        currentMarker &&
+        currentMarker.type &&
+        currentMarker.type === 'Preference'
```

Comment:
so, this doesn't seem to work now :) at least not in the example you provided.
My feeling (as said in the comments below) is that the `type` should indeed be `Preference` and then this code would actually work, but this is not what I see in the profile for now.

Reply:
Nope, you're right. That sample will no longer work. We changed the name in the gecko profiler patch to make it more obvious that these are preference reads and not some of those type of access. Because of that, the logic changed and older data captures won't work any more. I will post another test data file. Sorry!